### ISSUE TITLE
Add note about rootPath to docs

### ISSUE THREAD

**Issue by pmjones** [bookdown/Bookdown.Bookdown#16]
Per #15

**Comment by nasko** [comment]
Hello, where can I see sample usage of rootPath? Thanks!

**Comment by sandrokeil** [comment]
I'm working on an update for the docs. Use the develop branch for new features like tocDepth, CommonMark extensions and image download. Here is the example for _rootHref_:

``` json
{
  "title": "your title",
  "content": [
  ],
  "target": "./html",
  "rootHref": "http://example.com/"
}
```

**Comment by nasko** [comment]
Thanks for your timely response!

I wasn't initially aware how to achieve relative links, i.e. generate all the links such that when I move the target folder the links would still be valid. When I stumbled upon #15, I quickly decided that would work for me if I set `"rootHref": ""`, but it didn't - I saw in the code that when it was evaluated as empty it would default to '/', which was not ok for me. 

Now I tried `"rootHref": "./"` and it nailed it :)
